src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -32,47 +39,69 @@ type ToggleSettingOptionRowProps = {
 };
 const ICON_SIZE = 48;
 
-function ToggleSettingOptionRow({icon, title, subtitle, onToggle, subMenuItems, isActive, pendingAction, errors, onCloseError, disabled = false}: ToggleSettingOptionRowProps) {
+function ToggleSettingOptionRow({
+    key,
+    icon,
+    title,
+    subtitle,
+    shouldPlaceSubtitleBelowSwitch,
+    wrapperStyle,
+    onToggle,
+    subMenuItems,
+    isActive,
+    pendingAction,
+    errors,
+    onCloseError,
+    disabled = false,
+}: ToggleSettingOptionRowProps) {
     const styles = useThemeStyles();
 
+    const subTitleView = () => (
+        <Text
+            style={{
+                ...styles.textLabel,
+                ...(shouldPlaceSubtitleBelowSwitch ? styles.mt4 : styles.mt1),
+                ...(!shouldPlaceSubtitleBelowSwitch && styles.mr5),
+                ...styles.textSupporting,
+            }}
+        >
+            {subtitle}
+        </Text>
+    );
```

Comment:
This does not need to be a function. use `useMemo` to rerun a ready-to-render view